docs/_static/nuts-service-crypto.yaml

```diff
@@ -222,6 +222,32 @@ paths:
             text/plain:
               example:
                 "unknown legalEntity"
+  /crypto/csr/vendorca:
+    post:
+      summary: "Generate a CSR for requesting a vendor CA certificate."
+      operationId: generateVendorCACSR
+      tags:
+        - crypto
+      parameters:
```

Comment:
DN, otherName and domain extension?

Reply:
- DN is derived from 'name'; it's formatted as`CN=<name> CA,O=<name>,C=NL`. Do you think vendors should be able to decide on their own DN? I think we should at least require CA certificates to have the 'CA' postfix in the Common Name and the O component to hold the organization's name (with that name it's going to be registered in the registry).
- otherName = vendorID = identity, which is configured on the Nuts node. So no need to supply that.
- Since 'domain' is currently not really used and we only do healthcare, I don't think it adds something (except complexity) by adding it at this stage.

Reply:
Spec. added in certificate structure RFC